Client/Shared/UserProfile/ProfileDetails.razor

```diff
@@ -0,0 +1,85 @@
+﻿@if (_userProfileDetails is null)
+{
+    <LoadingIndicatorComponent />
+}
+else
+{
+    @if (_userProfileDetails.TopUsersPlacement is not null)
+    {
+        <span class="top-user @($"top-{_userProfileDetails.TopUsersPlacement}-user")">
+            <i class="icon-cup"></i>
+            @($"{_userProfileDetails.TopUsersPlacement}th best user")
```

Comment:
If user the best user, As I understand it will display 1th best user instead of 1st best user / The best user

Reply:
Yea I forgot about it